server/CVCMIServer.cpp

```diff
@@ -607,11 +608,11 @@ void handleLinuxSignal(int sig)
 }
 #endif
 
-int main(int argc, char** argv)
+int main(int argc, char * * argv)

```

Comment:
@dydzio0614 for something like ```***``` we should probably use boost::multi_array, but we really only have 13 matches for ``` * * ``` now.

@FeniksFire do you think we should still care about them?